### PR TITLE
new unbeelivable jelly candies

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Adds Jeleina Bjilka (Желейна Бджілка or Jelly Bee) jelly candies (inspired by Crazy Bee candies) to the Discount Den. 

**Media**

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Tunguso4ka
- add: new unbeelivable jelly candies flooded the Discount Den assortment

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml [hunk 1]
```diff
@@ -370,6 +370,21 @@
         - ReagentId: Theobromine
           Quantity: 3
 
+- type: entity
+  id: FoodSnackShalenaBjilka
+  parent: FoodSnackBase
+  name: Shalena Bjilka
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml:376]: should be `shalena bjilka`

File: Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml [hunk 2]
```diff
@@ -546,3 +561,13 @@
   components:
   - type: Sprite
     state: mre-wrapper
+
+- type: entity
+  noSpawn: true
+  parent: FoodPacketTrash
+  id: FoodPacketShalenaBjilkaTrash
+  name: empty Shalena Bjilka
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml:569]: same

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml (hunk 1 above)
Comment (Tunguso4ka) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml]: done.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml (hunk 2 above)
Comment (Tunguso4ka) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/snacks.yml]: done.

**Comment by lapatison** [comment]
Excuse me, but isn't that a registered trademark?
https://www.roshen.com/ro/en/products/boxed-sweets/crazy-bee-frutty

**Comment by Tunguso4ka** [comment]
> Excuse me, but isn't that a registered trademark? https://www.roshen.com/ro/en/products/boxed-sweets/crazy-bee-frutty

i forgor. Will rename ma candies.

**Comment by metalgearsloth** [comment]
The other food contents are generic but this just seems to be a specific real world food analogue being added? I'm not seeing a strong need to add this.

**Comment by moonheart08** [comment]
For those downvoting this PR solely because of political reasons i.e. the fact this is Ukranian: Your opinion has been discarded.

However, we're not adding niche regional foods at this time, and also just removed one, so this cannot be added either.

**Pull request closed without merging**